config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -65,6 +66,8 @@ presubmits:
               value: "6443"
             - name: PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER
               value: "true"
+            - name: AZUREDISK_CSI_DRIVER_VERSION
```

Comment:
Can we keep the settings needed for azure-disk csi drivers grouped and add a comment like the others in this section?

Do we need / want the csi driver running on the Windows nodes too?
if so we should add back in some of the env vars I dropped in the last PR
- WINDOWS=true
- WINDOWS_USE_HOST_PROCESS_CONTAINERS=true
- DISABLE_ZONE=true

Reply:
I believe we also need to set `DEPLOY_AZURE_CSI_DRIVER` based on

https://github.com/kubernetes/perf-tests/blob/8a1a241a3d4163b403faf110fe87c365a1d3817f/clusterloader2/run-e2e.sh#L56-L58

Reply:
We don't use the run-e2e.sh script from clusterloader2, it's the only place where that variable is used to install the Azure CSI Drivers